### ISSUE TITLE
v1 recipes cannot access global pinning of python_min

### ISSUE THREAD

**Issue by synapticarbors** [conda-forge/conda-smithy#2178]
It appears as if v1 recipes cannot access the global pinning of `python_min`. When re-rendering a recipe after using `crm` to generate a `recipe.yaml` and changing the `conda_build_tool` to `rattler-build`, re-rendering of a feedstock (or running `conda-smithy rerender` directly), results in the following error:

```
Adding in variants from /tmp/tmpf4uzuf10/conda-smithy/conda_build_config.yaml
Error:   × Failed to parse recipe

Error:   × Parsing: failed to parse match spec: unable to parse version spec: >=
    ╭─[24:7]
 23 │   run:
 24 │     - python >=${{ python_min }}
    ·       ─────────────┬────────────
    ·                    ╰── error parsing `python >=` as a match spec
 25 │     - pytorch >=2.0
    ╰────
```

The current work around is to set `python_min` explicitly in the `context`.

**Comment by wolfv** [comment]
The error is just with the `>=`. I thought we switched to a `3.11.*` syntax in the variant...

But in any case, this will be fixed by https://github.com/prefix-dev/rattler-build/pull/1122 that I am trying to finish over the course of the next few days

**Comment by jakirkham** [comment]
Also filed upstream issue: https://github.com/prefix-dev/rattler-build/issues/1248

The `>=` is for `requirements/run`, which only needs a lower bound

Another thing worth rechecking is whether re-rendering works with that fix. Had some issues getting `python_min` picked up in variant files in some cases

**Comment by wolfv** [comment]
The rattler-build issue is fixed in 0.32.0 which was just released.

**Comment by jakirkham** [comment]
Thanks Wolf! 🙏
